public/index.html

```diff
@@ -18,107 +22,292 @@
   <header role="banner" class="header">
     <div class="header__wrapper">
       <h1 class="header__primary">
-        <span class="header__primary--top">A Viget Exploration</span>
-        <img class="header__primary--btm icon" src="./src/icons/title.svg">
+        <span class="header__primary--top hdr-text">A Viget Exploration</span>
+        <img class="header__primary--btm" src="assets/images/title.svg" alt="improving medical adherence">
       </h1>
-
-      <img class="header__text-box__pills1 icon" src="src/icons/viget-logo.svg">
-      <img class="header__text-box__pills2 icon" src="src/icons/yellow-pill.svg">
-      <img class="header__text-box__pills3 icon" src="src/icons/more-pills.svg">
+      <img class="header__text-box__pills1" src="assets/images/viget-logo.svg" alt="pills">
+      <img class="header__text-box__pills2" src="assets/images/yellow-pill.svg" alt="a pill">
+      <img class="header__text-box__pills3" src="assets/images/more-pills.svg" alt="pills">
     </div>
   </header>
 
   <nav role="navigation" class="nav">
     <div class="nav__box">
-      <a class="nav__link" href="">The problem</a>
-      <a class="nav__link" href="">Key factors</a>
-      <a class="nav__link" href="">The question</a>
-      <a class="nav__link" href="">The solution</a>
+      <a class="nav__link--problem red-nav" href="#problem-section">The problem</a>
+      <a class="nav__link--keyFactors red-nav" href="#keyFactors-section">Key factors</a>
+      <a class="nav__link--question red-nav" href="#question-section">The question</a>
+      <a class="nav__link--solution blue-nav" href="#solution-section">The solution</a>
     </div>
   </nav>
 
-  <section role="region" class="problem">
+  <section id="problem-section" class="problem">
     <div class="problem__wrapper">
-      <p class="problem__intro">Patients who don’t take medication as prescribed are at risk of worsening health, increased hospitalization, and a greater chance of mortality.
+      <h2 class="screen-reader">The problem</h2>
+      <p class="problem__intro intro-text">Patients who don’t take medication as prescribed are at risk of worsening health, increased hospitalization, and a greater chance of mortality.
       </p>
```

Comment:
This indentation is a little odd. I'd do this instead:

```
<p>
  Text here
</p>
```
Just a preference. This is a more nit-picky note.

Reply:
Changed, but why is it odd?  
Is it because its a long text? Do you prefer to do this for the rest of the <p> tags?

Reply:
Folks usually take one of two approaches with `<p>` content. Either:

```
<p>
  Text here
</p>
```
OR:

```
<p>Text here</p>
```

Yours are somewhere in the middle with the end tag on a new line. There's nothing particularly wrong with it, it's just not what I'm used to seeing.

I personally prefer the first since it's easy to make out where the end tag falls and you can read any classes/properties clearly. On your version, it's hard to tell when scanning where the classes/attributes end and the content begins.